doc/cug.tex

```diff
@@ -3281,6 +3265,37 @@ \subsubsection{Graph Types}
 \label{fig-satellite}
 \end{figure}
 
+\subparagraph{Advanced Integer Cycling Syntax}
+
+The same syntax used to reference the initial and final cycle points
+introduced in~\ref{referencing-the-initial-and-final-cycle-points}) for
+use with datetime cycling can also be used for integer cycling. For
+example you can write:
+
+\lstset{language=suiterc}
+\begin{lstlisting}
+[[[ R1/^ ]]]     # Run once at the initial cycle point
+[[[ R1/$ ]]]     # Run once at the final cycle point
+[[[ R3/^/P2 ]]]  # Run three times with step two starting at the
+                 # initial cycle point
+\end{lstlisting}
+\lstset{language=transcript}
+
+Likewise the syntax introduced in~\ref{excluding-dates} for excluding
+a particular point from a recurrence also works for integer cycling
+meaning you can write:
+
+\lstset{language=suiterc}
+\begin{lstlisting}
+[[[ R/P4!8 ]]]       # Run with step four, to the final cycle point
+                     # but not at point 8
+[[[ R3/3/P2!5 ]]]    # Run with step two from point 3 but not at
+                     # point 5
+[[[ R/+P1/P6!14 ]]]  # Run with step six from one step after the
+                     # initial cycle point but not at point 14
+\end{lstlisting}
+\lstset{language=transcript}
+
```

Comment:
Inconsistent use of english vs digit numbers (just use digits?) - i.e. "step 2" rather than "step two"?